server/apiRoutes/event.js

```diff
@@ -64,7 +67,7 @@ router.get('/events', async(req, res, next) => {
 //get all attendees for an event
 router.get('/events/:id', async(req, res, next) => {
     try {
-        const event = Event.findOne({ where: {id: req.params.id} });
+        const event = await Event.findOne({ where: {id: req.params.id} });
         const eventAttendees = await event.getUsers();
         res.send(eventAttendees);
```

Comment:
One thing about this that I think will need to be fixed is how much user information do we want to send to the front end in this method? Maybe we think about adding some attributes to the event.getUsers() call.